tests/testthat/test-derive_params_growth_age.R

```diff
@@ -31,7 +31,8 @@ test_that("derive_params_growth_age Test 1: derive_params_growth_age works", {
 
   expected <- c(
     ((10 / 2)^1 - 1) / (1 * 3),
-    ((100 / 8)^7 - 1) / (7 * 9)
+    ((11 / 5)^4 - 1) / (4 * 6),
```

Comment:
can we add a fake example with metadata say 27 and 28 and the age as 27.5 to show this rounds down to 27 as nearest then in keeping with R default rounding rules for .5 values?